app/src/main/res/values/group_setting_style.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="groups_number_setting">
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:imeOptions">actionDone</item>
+        <item name="android:singleLine">true</item>
+        <item name="android:gravity">center</item>
+        <item name="android:inputType">number</item>
+        <item name="android:layout_marginLeft">5dp</item>
+        <item name="android:paddingTop">3dp</item>
+        <item name="android:paddingBottom">3dp</item>
+        <item name="android:paddingLeft">10dp</item>
+        <item name="android:paddingRight">10dp</item>
+        <item name="android:background">@drawable/text_input_border</item>
+        <item name="android:textSize">16dp</item>
+        <item name="android:maxLength">4</item>
```

Comment:
I made the length to 4 for the numOfGroups and keep it to 2 for the namesPerGroup

Reply:
Ah, 2 is consistent with the choosing page. I feel like it's weird to have inconsistency; we should just cap to 2 in all places. With 4 digits, you need to deal with the EditText getting very large.